content-security-policy/embedded-enforcement/subsumption_algorithm-hashes.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Embedded Enforcement: Subsumption Algorithm - Hashes.</title>
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <script src="../support/testharness-helper.sub.js"></script>
+</head>
+<body>
+  <script>
+    var tests = [
+      { "name": "'sha256-abc123' is properly subsumed.", 
+        "required_csp": "style-src http://example1.com/foo/ 'self' 'unsafe-hashed-attributes' 'strict-dynamic' 'sha256-abc123'", 
```

Comment:
Nit: Can you move one of the simpler tests up? :) It would be nice to start with a basic example before dealing with strict-dynamic and lists and etc. :)